.github/workflows/publish_openrpc_document.yml

```diff
@@ -1,10 +1,8 @@
 name: Publish OpenRPC document
 
 on:
-  push:
-    tags:
-      - 'v*'
-  workflow_dispatch:
+  release:
```

Comment:
I'd add a comment that it needs to be in release bc the artifact uploading section.

Reply:
Could you clarify this?

Reply:
This workflow can never work with tags since tags are created before having a release and since the last step of this workflow uploads artifacts to the release, the tag approach won't ever work